### DISCUSSION TITLE
Why don't Payload integrated FE ?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [payloadcms/payload#1783] [Q&A] [answered]
I saw PayloadCMS video on Youtube
But why don't you add FE too ? We can do all in one like NextJS (FE and BE)
Example: abc.com for FE
and abc.com/admin for BE ?
I just wonder why.

**Comment by jacobsfletch** [comment, marked as answer]
@IRediTOTO this is totally possible, and really just a matter of preference. One way to do this is to configure Payload to serve your FE by sharing the Express server. Here's an example of how this is done: https://github.com/payloadcms/nextjs-custom-server
